ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayKml/DisplayKml.h

```diff
@@ -0,0 +1,62 @@
+// [WriteFile Name=DisplayKml, Category=Layers]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef DISPLAYKML_H
+#define DISPLAYKML_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class SceneQuickView;
+class Scene;
+class KmlLayer;
+class KmlDataset;
+class PortalItem;
+}
+}
+
+#include "Viewpoint.h"
+#include <QQuickItem>
+
+class DisplayKml : public QQuickItem
+{
+  Q_OBJECT
+
+public:
+  explicit DisplayKml(QQuickItem* parent = nullptr);
+  DisplayKml::~DisplayKml() = default;
```

Comment:
Can move this into the cpp file.

Reply:
Agreed this can be declared here as usual but `DisplayKml::~DisplayKml() = default;` should go in the cpp file.